iroh-test/Cargo.toml

```diff
@@ -0,0 +1,14 @@
+[package]
+name = "iroh-test"
+version = "0.5.1"
+edition = "2021"
+readme = "README.md"
+license = "MIT OR Apache-2.0"
+authors = ["n0 team"]
+repository = "https://github.com/n0-computer/iroh"
+publish = false
```

Comment:
I believe we have to publish it, otherwise we can't publish the other crates, dev dependencies need to be available

Reply:
oh, that's sad

Reply:
fixed